server/routes/support/handlers/offenderDetail.ts

```diff
@@ -18,7 +23,33 @@ export default class OffenderDetailRoutes {
       ? await this.communityService.getStaffDetailByStaffCode(probationPractitioner?.staff.code)
       : undefined
     const hdcStatus = _.head(await this.prisonerService.getHdcStatuses([prisonerDetail], user))
+    const licenceSummary = await this.licenceServer.getLatestLicenceByNomisIdsAndStatus([nomsId], [], user)
 
+    let licencecrn = 'Not found'
+    let crd = 'Not found'
+    let releaseDate = 'Not found'
+    let sentenceStartDate = 'Not found'
+    let sed = 'Not found'
+    let led = 'Not found'
+    let tussd = 'Not found'
+    let licensetused = 'Not found'
+
+    if (licenceSummary != null) {
+      const licence = await this.licenceServer.getLicence(licenceSummary.licenceId.toString(), user)
+
+      licencecrn = licence.crn
+      crd = licence.conditionalReleaseDate
+      releaseDate = licence.actualReleaseDate
+      sentenceStartDate = licence.sentenceStartDate
+      sed = licence.sentenceEndDate
+      led = licence.licenceExpiryDate
+      tussd = licence.topupSupervisionStartDate
+        ? moment(licence.topupSupervisionStartDate).format('DD MMM YYYY')
```

Comment:
This has the DD MMM YYYY format, but if you tried to access it I imagine it will likely fail and raise an error, as the dates stored on the licence are in `dd/mm/yyyy` format, and moment expects them in `mm/dd/yyyy` format. This means that something like `13/1/1970` would be evaluated as 1st of the 13th month, raising an error.
You need to specify `moment(datestring, 'DD/MM/YYYY')` for it to parse correctly.